include/openmc/capi.h

```diff
@@ -110,6 +110,7 @@ extern "C" {
   int openmc_tally_get_nuclides(int32_t index, int** nuclides, int* n);
   int openmc_tally_get_scores(int32_t index, int** scores, int* n);
   int openmc_tally_get_type(int32_t index, int32_t* type);
+  int openmc_tally_get_writeable(int32_t index, bool* writeable);
```

Comment:
"writeable" is not a word where "writable" is:
```suggestion
  int openmc_tally_get_writable(int32_t index, bool* writable);
```

Reply:
:man_facepalming: